projects/module-1/colorandom.md

```diff
@@ -46,76 +46,73 @@ Then, as a team:
 
 ## Progression
 
-### Iteration 0 - HTML & CSS 
+### Iteration 0 - Basic Layout
 
-<img src="./assets/colorandom/initial-view.png" alt="Initial view">
+![iteration 0](./assets/colorandom/iteration0.png)
 
 - When the page loads, we should see:
   -  a title
-  -  5 colors labeled with their hex codes, with an unlocked lock emoji next to each hex code
-      -  Hardcode these for now
-  -  2 buttons (one to get a new random palette, one to save the displayed palette)
-  -  a section for displaying saved palettes (later iteration)
+  -  5 colors labeled with their hex codes, hardcoded to match the colors in the screenshot above for now
 
-The CSS in this iteration should mostly focus on LAYOUT (positioning), rather than STYLE (fonts and colors).
+You should be matching the comp closely. Act like you're a frontend dev on a team and this is the design you've been given to match. Details and spacing matter! Comp notes:
+  - Background color: `#E7F1F1`
+  - Font: [Poppins](https://fonts.google.com/specimen/Poppins?query=poppins)
```

Comment:
Love that students are getting practice matching a comp and getting practice with those design patterns!